src/nu/nerd/nerdmoods/NerdMoods.java

```diff
@@ -131,29 +137,27 @@ protected void cmdPtime(CommandSender sender, String[] args) {
             showUsage = true;
         }
 
-        if (showUsage) {
-            sender.sendMessage(ChatColor.RED + "Usage: /ptime [day|night|<time>]");
+        if (showUsage || time < 0) {
+            sender.sendMessage("Usage: /time [day|night|<number>]");
             return;
         }
 
-        if (time < 0) {
-            time += 24000;
-        }
-
+        world.setTime(time);
```

Comment:
This changes the entire server's time, not the player's local time. This can be confirmed by summoning a zombie with `/ptime night` toggled.